core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -583,8 +601,9 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
               .result
           )
         )
+        // measure the byte size of the actual value is exactly what we specified
         assertResult(EntityComponent.allAttributeValuesColumnSize) {
-          entityRec.get.allAttributeValues.get.length
+          entityRec.get.allAttributeValues.get.getBytes(StandardCharsets.UTF_8).length
         }
       }
```

Comment:
I verified that this test fails as `java.sql.BatchUpdateException: Data truncation: Data too long for column 'all_attribute_values' at row 1` without the runtime changes in `EntityComponent`